src/platform-integration/windows/extern_win.md

```diff
@@ -0,0 +1,39 @@
+---
+title: External windows in Flutter Windows apps
+description: Special considerations for adding external windows to Flutter apps
+---
+
+# Windows lifecycle
+
+## Who is affected
+
+Windows applications built against Flutter version > 3.13 that open non-Flutter windows.
+
+## Overview
+
+When adding a non-Flutter window to a Flutter Windows app, it will not be part
+of the logic for application lifecycle state updates by default. For example,
+this means that when the external window is shown or hidden, the app lifecycle
+state will not appropriately update to inactive or hidden. As a result, the app
+may receive incorrect lifecycle state changes through
+[WidgetsBindingObserver.didChangeAppLifecycle][].
+
+# What do I need to do?
+
+In order to add the external window to this application logic, the window's
+`WndProc` procedure must invoke `FlutterEngine::ProcessExternalWindowMessage`.
+
+The following may be added to a window message handler function to achieve this:

```

Comment:
**chore (non-blocking):** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
To achieve this, add the following code to a window message handler function:
```